diff --git a/dist/index.js b/dist/index.js
index 16daa15df86bfa40062889c8bec02c8881354a18..952966a5b6f1a3f075fcc242e7a75d3f7596db54 100755
--- a/dist/index.js
+++ b/dist/index.js
@@ -33089,7 +33089,7 @@ var AgentMode = class _AgentMode {
     {
       type: "workspaceWrite",
       writableRoots: [],
-      networkAccess: false,
+      networkAccess: true,
       excludeTmpdirEnvVar: false,
       excludeSlashTmp: false
     },
