diff --git a/dist/index.js b/dist/index.js
index f15585a172e7443da1235ad8beb2509cc4925ae7..4f523fbfb17d6e35ed7982e692119a050d8f6045 100755
--- a/dist/index.js
+++ b/dist/index.js
@@ -24207,7 +24207,7 @@ var AgentMode = class _AgentMode {
     {
       type: "workspaceWrite",
       writableRoots: [],
-      networkAccess: false,
+      networkAccess: true,
       excludeTmpdirEnvVar: false,
       excludeSlashTmp: false
     },
